CONTRIBUTING.rst

```diff
@@ -114,7 +114,15 @@ To base a branch directly off of `dev` instead of `master`, create a new one as
     $ flake8 janitor tests
     $ py.test
 
-   flake8 and pytest are instaled when you create the development environment.
+   flake8 and pytest are installed when you create the development environment.
+
+   When you run the test locally, these test are automatically skipped if you don't have the optional dependencies (e.g. ``rdkit``) installed:
```

Comment:
@SorenFrohlich, actually, on second review, I think it may be more maintainable and compact if we change this line to:

```suggestion
   When you run the test locally, the tests in ``chemistry.py`` are automatically skipped if you don't have the optional dependencies (e.g. ``rdkit``) installed.
```

And then we can delete the rest of the lines. Doing so makes this more maintainable.